server/models/post.model.js

```diff
@@ -163,43 +164,54 @@ PostSchema.statics = {
     if (type === 'top') {
       sort = { score: -1 };
     }
-    const queryPromise = this.find(query, 'content title date mp3 link score featuredImage upvoted downvoted tags categories')
+    const queryPromise = this.find(query, this.defaultSelected)
```

Comment:
I am just curious what defaultSelected is, I did a quick search for it but didn't see it in the mongoose docs.

Reply:
Out of context of overall usage this change doesn't make sense - and maybe it should be a separate commit but it is quite related to the change. Basically, I wanted the favorite/bookmark's `listBookmarkedPostForUser` function to return data of the same shape as post's `list` function. So this includes the same `select` param for `find`. So instead of repeating the same argument, I moved it to a `defaultSelected` static within the `PostSchema` and referenced it in both functions. It likely would be better put at the top of the static list... if you agree I'll change that (also need to add /:userId/ option)

There may be a better way to achieve this kind of re-use... but my feeling is this is a decent stop-gap.

Reply:
Ah, I see. I think this works. Which static list are you referring to?

Reply:
`defaultSelected` is a static attribute on `PostSchema`. So you can get this with either `this.defaultSelected` in other `PostSchema` static methods, or `Post.defaultSelected` if post is required.

Maybe there is a better name for it, defaultSelectedFields so it is explicit?

Also, I didn't implement /users/:id/bookmarked, only /users/me/bookmarked. I can't quite recall why, but if everything else is fine, I can do this tomorrow morning for you to accept the PR.